env0/utils.go

```diff
@@ -134,6 +119,54 @@ func readResourceData(i interface{}, d *schema.ResourceData) error {
 	return nil
 }
 
+func readResourceDataSliceStructHelper(field reflect.Value, resource interface{}) error {
```

Comment:
This is already tested in:
TestReadResourceDataModule - for slice of structs.
TestReadResourceDataNotificationProjectAssignment - for slice of strings.